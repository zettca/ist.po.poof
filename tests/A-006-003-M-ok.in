1
6
1
a
a
1
z
z
1
d
d
1
rr
rr
1
r
r
1
aa
a
2
0
0
