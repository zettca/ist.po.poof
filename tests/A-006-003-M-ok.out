Menu Principal
1 - Novo
2 - Abrir
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Identificador do utilizador: Nome do utilizador: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: a:a:/home/a
aa:a:/home/aa
d:d:/home/d
r:r:/home/r
root:Super User:/home/root
rr:rr:/home/rr
z:z:/home/z
Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: 